index.php

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <?php $base = "../../" ?>
+    <base href="../../">
+    <script src="js/jquery-2.2.4.min.js"></script>
+    <script src="js/facebox.js"></script>
+    <script src="js/gameSettings.js"></script>
+    <link rel="stylesheet" type="text/css" href="css/facebox.css"/>
+    <link rel="stylesheet" type="text/css" href="css/main.css"/>
+    <link rel="stylesheet" type="text/css" href="css/bootstrap.css"/>
+    <script type="text/javascript">
+        jQuery(document).ready(function($) {
+            $('a[rel*=facebox]').facebox()
+        })
+    </script>
+</head>
+<body>
+<div class="container">
+    <?php include $base."header.php"; ?>
+    <nav>
+        <ul>
+        <li><a href="">Home</a></li>
+        </ul>
+        <?php include $base."leftMenuGame.php"; ?>
+    </nav>
+    <article>
+        <h1 id="gameName">Word-Finder</h1>
+        <h3 id="groupName">The Raging Dissapointments</h3>
+        <h3>Instructions:</h3>
+        <div id="gameDesc" class="jumbotron">
+            <div>
+                <p>
+                    <b>How to Win</b>
+                    <br>
+                    The goal of the game is to have the most points when the game ends. The game ends when all rounds have been completed. A round being a cycle where each player gets a single turn.
```

Comment:
"The goal of the game is to accumulate the most points by the time the game finishes. The game has <code>n</code> rounds (defined by the player). A round consists of a turn for each player."

Reply:
the n is highlighted because I wrapped it in the `<code>` tag

Reply:
Added the changes to the file.